docs/user/discussion.rst

```diff
@@ -0,0 +1,150 @@
+These posts can only be created on the course discussion overview which is showed in the screenshot below.
+
+The ``Discussion`` space of an Artemis course serves as overview for *all* posts in a course.
+Hence, course-wide posts as well as exercise posts and lecture posts are listed.
+Here, users can easily query, sort and filter existing posts.
+
+|course-posts|
+
+Features for Users
+------------------
+
+This section captures the most important features that are offered to any Artemis user.
+
+Search, Filter and Sort Posts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On the course discussion overview, a user can query *all* existing posts by different criteria.
+This can be a text-based search, context filters (i.e., a certain lecture, exercise or course-wide topic), or other post characteristics such as if it is already resolved.
+By using different sort configurations, users can find the information they are looking for at the top of the list: for example the post that got the most votes, or the post that was created most recently.
+
+React on Posts
+^^^^^^^^^^^^^^
+
+To foster interaction we integrate the well-known emoji reaction bar.
+Each user in the course can react on every post by making use of the emoji selection button.
+The ``+`` emoji serves as up-voting reaction, which influences the display order.
+
+Reference Posts
+^^^^^^^^^^^^^^^
+
+If users want to refer to other posts, they can integrate a simple pattern including the hashtag (``#``) combined with the post identifier.
+A post's identifier is appended to the post title (such as seen in the screenshot).
+
+When clicking a reference used in a post's text, the user is navigated to the referenced post.
+
+Prevent Duplicated Posts
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to prevent duplicated questions from being posted, we integrate a duplication check that runs during post creation.
+We strongly recommend users that create a post, to check the automatically provided list of similar posts to find out if the question in mind has already been asked and resolved in the best case.
+
+Mark Your Post As Resolved
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Marking a post as resolved will indicate to other users that the posted question is resolved and does not need any further input.
+This can be done by clicking the checkmark next to the answer post.
+Note, that only the author of the post as well as a moderator can perform this action.
+This is helpful for moderators to identify open questions, e.g., by applying the according filter in the course overview.
+It also highlights the correct answer for other students that have a similar problem and search for a suitable solution.
+
+Tag Your Post
+^^^^^^^^^^^^^
+
+When creating a post, users can choose to add arbitrary tags.
+Tagging a post will further narrow down the post purpose or content in precise and descriptive buzzwords, that might follow a course-specific taxonomy.
```

Comment:
```suggestion
Tagging a post will further narrow down the post purpose or content in precise and descriptive keywords, that might follow a course-specific taxonomy.
```
‘Buzzword’ has the fairly negative connotation of being overused. I don’t think this meaning is intended here.

Reply:
changed, thanks!